dev-tools/phpstan/baseline/missingType.property.php

```diff
@@ -0,0 +1,67 @@
+    'count' => 1,
+    'path' => __DIR__ . '/../../../src/Console/Command/CheckCommand.php',
+];
+$ignoreErrors[] = [
+    'message' => '#^Property PhpCsFixer\\\\Console\\\\Command\\\\CheckCommand\\:\\:\\$defaultName has no type specified\\.$#',
+    'count' => 1,
+    'path' => __DIR__ . '/../../../src/Console/Command/CheckCommand.php',
+];
+$ignoreErrors[] = [
+    'message' => '#^Property PhpCsFixer\\\\Console\\\\Command\\\\DescribeCommand\\:\\:\\$defaultName has no type specified\\.$#',
+    'count' => 1,
+    'path' => __DIR__ . '/../../../src/Console/Command/DescribeCommand.php',
+];
+$ignoreErrors[] = [
+    'message' => '#^Property PhpCsFixer\\\\Console\\\\Command\\\\DocumentationCommand\\:\\:\\$defaultName has no type specified\\.$#',
+    'count' => 1,
+    'path' => __DIR__ . '/../../../src/Console/Command/DocumentationCommand.php',
+];
+$ignoreErrors[] = [
+    'message' => '#^Property PhpCsFixer\\\\Console\\\\Command\\\\FixCommand\\:\\:\\$defaultDescription has no type specified\\.$#',
+    'count' => 1,
+    'path' => __DIR__ . '/../../../src/Console/Command/FixCommand.php',
+];
+$ignoreErrors[] = [
+    'message' => '#^Property PhpCsFixer\\\\Console\\\\Command\\\\FixCommand\\:\\:\\$defaultName has no type specified\\.$#',
+    'count' => 1,
+    'path' => __DIR__ . '/../../../src/Console/Command/FixCommand.php',
+];
+$ignoreErrors[] = [
+    'message' => '#^Property PhpCsFixer\\\\Console\\\\Command\\\\HelpCommand\\:\\:\\$defaultName has no type specified\\.$#',
+    'count' => 1,
+    'path' => __DIR__ . '/../../../src/Console/Command/HelpCommand.php',
+];
+$ignoreErrors[] = [
+    'message' => '#^Property PhpCsFixer\\\\Console\\\\Command\\\\ListFilesCommand\\:\\:\\$defaultName has no type specified\\.$#',
+    'count' => 1,
+    'path' => __DIR__ . '/../../../src/Console/Command/ListFilesCommand.php',
+];
+$ignoreErrors[] = [
+    'message' => '#^Property PhpCsFixer\\\\Console\\\\Command\\\\ListSetsCommand\\:\\:\\$defaultName has no type specified\\.$#',
+    'count' => 1,
+    'path' => __DIR__ . '/../../../src/Console/Command/ListSetsCommand.php',
+];
+$ignoreErrors[] = [
+    'message' => '#^Property PhpCsFixer\\\\Console\\\\Command\\\\SelfUpdateCommand\\:\\:\\$defaultName has no type specified\\.$#',
+    'count' => 1,
+    'path' => __DIR__ . '/../../../src/Console/Command/SelfUpdateCommand.php',
+];
+$ignoreErrors[] = [
+    'message' => '#^Property PhpCsFixer\\\\Console\\\\Command\\\\WorkerCommand\\:\\:\\$defaultDescription has no type specified\\.$#',
+    'count' => 1,
+    'path' => __DIR__ . '/../../../src/Console/Command/WorkerCommand.php',
+];
+$ignoreErrors[] = [
+    'message' => '#^Property PhpCsFixer\\\\Console\\\\Command\\\\WorkerCommand\\:\\:\\$defaultName has no type specified\\.$#',
+    'count' => 1,
+    'path' => __DIR__ . '/../../../src/Console/Command/WorkerCommand.php',
+];
```

Comment:
E.g. in this file all errors are very similar (command name and description).
This is more noticeable due to the separate file for this error type.

So I might have a look to see if I can find a solution for that issue. I guess there is a bc reason for these errors. So maybe the solution is to just add an ignore rule to the main phpstan config file for now.